pkg/tfbridge/info/info.go

```diff
@@ -441,9 +441,15 @@ type Schema struct {
 	// a name to override the default when targeting C#; "" uses the default.
 	CSharpName string
 
-	// a type to override the default; "" uses the default.
+	// An optional Pulumi type token to use for the Pulumi type projection of the current property. When unset, the
+	// default behavior is to generate fresh named Pulumi types as needed to represent the schema. To force the use
+	// of a known type and avoid generating unnecessary types, use both [Type] and [OmitType].
 	Type tokens.Type
 
+	// Used together with [Type] to omit generating any Pulumi types whatsoever for the current property, and
+	// instead use the object type identified by the token setup in [Type].
+	OmitType bool
```

Comment:
Let's constrain our value space as much as possible here:
```suggestion
	// Used together with [Type] to omit generating any Pulumi types whatsoever for the current property, and
	// instead use the object type identified by the token setup in [Type].
	//
	// It is an error to set [OmitType] to true without specifying [Type].
	OmitType bool
```